contrib/linux/actions/traceroute.yaml

```diff
@@ -19,3 +19,5 @@ parameters:
     description:  No. of quereies to each hop
     default: 3
     position: 3
+  hosts:
```

Comment:
Wait, there are two arguments now - host and hosts?

And other arguments are positional and this one is not - don' think this will work well :P